docroot/modules/custom/va_gov_backend/src/EventSubscriber/EntityEventSubscriber.php

```diff
@@ -293,6 +294,25 @@ public function lockTitleEditing(array &$form, FormStateInterface $form_state) {
     }
   }
 
+  /**
+   * Locks down API Id for non-admins.
+   *
+   * @param array $form
+   *   The form.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The form state.
+   */
+  public function lockApiIdEditing(array &$form, FormStateInterface $form_state) {
+    $form_object = $form_state->getFormObject();
+    $bundle = NULL;
+    if ($form_object instanceof ContentEntityForm) {
+      $bundle = $form_object->getEntity()->bundle();
+    }
+    if (!$this->userPermsService->hasAdminRole() && $bundle === 'health_care_service_taxonomy') {
```

Comment:
This should only be for true admins.
```suggestion
    if (!$this->userPermsService->hasAdminRole(TRUE) && $bundle === 'health_care_service_taxonomy') {
```

Reply:
Ty for catching that, @swirtSJW :heart: